docs/supported-technologies.asciidoc

```diff
@@ -69,6 +70,15 @@ We support these tornado versions:
  * 6.0+
 
 
+[float]
+[[supported-sanic]]
+=== Sanic
+
+We support these sanic versions:
+
+ * 20.12.2+
```

Comment:
I'm not very familiar with the Sanic community and how often people upgrade to the newest version. In your opinion, is it an acceptable compromise to only support the currently latest version?

Reply:
Sanic has an LTS train. So, I would prefer to keep the LTS + the head of the branch release under the supported path if that is alright with you and the community under `elastic-apm`